hummingbot/connector/exchange/kucoin/kucoin_exchange.py

```diff
@@ -812,14 +812,16 @@ async def _update_trading_fees(self):
             throttler=self._throttler,
             time_synchronizer=self._time_synchronizer) for trading_pair in self._trading_pairs]
 
-        params = {"symbols": ",".join(trading_symbols)}
-
-        resp = await self._api_request(
-            path_url=CONSTANTS.FEE_PATH_URL,
-            params=params,
-            method=RESTMethod.GET,
-            is_auth_required=True,
-        )
+        resp = {"data": []}
+        for pair in trading_symbols:
+            params = {"symbols": pair}
+            r = await self._api_request(
+                path_url=CONSTANTS.FEE_PATH_URL,
+                params=params,
+                method=RESTMethod.GET,
+                is_auth_required=True,
+            )
+            resp["data"] += r["data"]
```

Comment:
The issue that is causing the error when querying fees for multiple trading pairs should be fix instead of turning the single request into multiple requests. The multiple requests will impact the rate limits consumption and could cause the server to stop responding to the client.